### DISCUSSION TITLE
rucsoundings.noaa.gov = 502 Bad Gateway

### DISCUSSION THREAD

**Discussion by sators** [NOAA-GSL/GSL-Regional-Model-Forum#42] [Q&A] [answered]
Hi - I'm trying to access the soundings site at https://rucsoundings.noaa.gov/ across multiple devices and multiple internet connections and all are getting `502 Bad Gateway`.  Is the site down?  Thanks!

**Comment by aeharding** [comment]
Yeah, it's down since at least yesterday. I've been using backup api (open-meteo)

**Comment by BillMoninger** [comment, marked as answer]
Unfortunately, we had to take the site down due to a security vulnerability. We believe we've fixed the vulnerability, but, this being a Friday, I'm afraid the site is likely to remain down over the weekend.

**Reply by BillMoninger** [reply]
I'm happy to report our site https://rucsoundings.noaa.gov/ is back up!  Thanks to our security and IT staff for a quick response.

**Reply by sators** [reply]
Thank you much!! 🎉
